pkg/auth/handler/webapp/static_assets.go

```diff
@@ -60,15 +60,13 @@ func (h *StaticAssetsHandler) Open(name string) (http.File, error) {
 		return nil, os.ErrNotExist
 	}
 
-	resolvePath := filePath
-
 	// Use GlobalEmbeddedResourceManager to check if the static asset is belong to it
-	if asset, ok := h.EmbeddedResources.Resolve(resolvePath); ok {
+	if asset, ok := h.EmbeddedResources.Resolve(filePath, hashInPath); ok {
```

Comment:
The new manager need not to support hash checking. It should resolve `p` directly.